tests/snapshots/add/target.out/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
+
+[package]
+name = "cargo-list-test-fixture"
+version = "0.0.0"
+
+[target]
+
+[target.i686-unknown-linux-gnu]
```

Comment:
Wanted to call out that I have not gotten to this item yet.  I wanted to update this PR with where I was at rather than waiting on any more changes.

Also note that this will only happen when creating the table from scratch.  When the hidden table already exists, we won't change its visibility.